scripts/semver.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+###
+# semantic version comparition using semver specification http://semver.org/
+# This bash script compares pre-releases alphabetically as well
+#
+# returns 1 when A greater than B
+# returns 0 when A equals B
+# returns -1 when A lower than B
+#
+# Usage
+# chmod +x semver.sh
+# ./semver.sh 1.0.0 v1.0.0-rc.0
+# --> 1
+#
+# Author Ariel Rodriguez
+# License MIT
+###
+semver_compare() {
+  local version_a version_b pr_a pr_b
+  # strip word "v" and extract first subset version (x.y.z from x.y.z-foo.n)
+  version_a=$(echo "${1//v/}" | awk -F'-' '{print $1}')
+  version_b=$(echo "${2//v/}" | awk -F'-' '{print $1}')
+
+  if [ "$version_a" \= "$version_b" ]
+  then
+    # check for pre-release
+    # extract pre-release (-foo.n from x.y.z-foo.n)
+    pr_a=$(echo "$1" | awk -F'-' '{print $2}')
+    pr_b=$(echo "$2" | awk -F'-' '{print $2}')
+
+    ####
+    # Return 0 when A is equal to B
+    [ "$pr_a" \= "$pr_b" ] && echo 0 && return 0
+
+    ####
+    # Return 1
+
+    # Case when A is not pre-release
```

Comment:
I’d remove `Case` from the beginning. `When A is not a pre-release` is suffiecient (also note the indefinite article `a` before `pre-release`).